pkg/utils/common.go

```diff
@@ -26,3 +27,13 @@ func SliceMap[T any, U any](in []T, f MapFunc[T, U]) []U {
 	}
 	return out
 }
+
+func SliceFilter[T any](in []T, f FilterFunc[T]) []T {
```

Comment:
Isn't this part of the lo library as Filter?

https://github.com/samber/lo

Reply:
mikhail-aws suggested to remove `lo` package
https://github.com/aws/aws-application-networking-k8s/pull/363#discussion_r1308037786